config/default/common/config/metadata/stories/dust_storms_overview_2019/step001.md

```diff
@@ -0,0 +1,3 @@
+Commonly occurring in arid and semi-arid regions like deserts, sand and dust storms occur when strong winds pick up dust and sand from areas with dry, bare soils, lift the dust into the atmosphere, and transports the dust many, many kilometers away. Main sources of the dust are Northern Africa, the Arabian Peninsula, Central Asia and China. Areas like Australia, America and South Africa have minor contributions, yet are still important. Providing a general idea of where the major dust contributors are, this Dust Surface Mass Concentration layer from the Modern-Era Retrospective analysis for Research and Applications, Version 2 (MERRA-2) is displaying the dust surface mass concentrations for February 2019. MERRA-2 assimilates space-based observations and model-bases analyses to produce long-term, global information on the Earth System.
```

Comment:
Maybe change to "Sand and dust storms commonly occur in arid and semi-arid regions, like deserts, when strong winds pick up...." instead of using the word occur twice.

Reply:
Thanks for the feedback - I have made some changes!